### PR TITLE
19 render inputtasks in box task

### LINKED ISSUE #19: Scripting: View all current Tasks in local storage in the Tasks Box
- Initial idea: loop
- create it as function because we may loop again later after each delete/edit.
- make sure it run at the begining to create current tasks in local storage

- [ ] in HTML: Need to replace due-date as `<label>` to  `<input  class="due-date" readonly>`
- [ ] in CSS: Need to add space between the elements of section of `class="right"` (make space between so user not mistakely press not needed button).
- [ ] in JS: we need to add the unique number in the loop function as a `<lablel>` element in the task and it is hidden element (in css `display : none`)
so we can connect edit/save/remove/done in window to the array.
> Note:
check Css for the task after adding `<input>` insted of `<label>` (it is longer and may affect the buttons in right section)

### PR DESCRIPTION
**Pull request by FatmaHossam5**

### CODE REVIEW

File: script/main.js [hunk 1]
```diff
@@ -54,3 +56,61 @@ add.onclick = function () {
...
+    left.append(checkBox);
+
+    let name_sec = document.createElement("section");
+    let name = document.createTextNode(element.name);
+    name_sec.className = "task-name";
+    name_sec.append(name);
+
+    
+    let due_sec = document.createElement("section");
+    let due = document.createTextNode(element.due_time);
```
Comment (MohammedHasan-99) [inline script/main.js]: make due as input type instead:
- let due = document.createElement("input");
- due.type = "datetime-local";
- due.value = element.due_time;
- due.readOnly = true;

File: script/main.js [hunk 2]
```diff
@@ -54,3 +56,61 @@ add.onclick = function () {
...
+    icon2.className = "task-icon fas fa-trash";
+    remove.append(icon2);
+
+
+    right.append(edit,remove);
+
+    task.append(left,right);
+
+    listDiv.appendChild(task);
+    console.log(element);
```
Comment (MohammedHasan-99) [inline script/main.js]: delete this line :3
line 114

File: css/home.css [hunk 3]
```diff
@@ -48,7 +48,7 @@ body {
 
 #addTaskButton {
   color: #4044ca;
-  background-color: white;
```
Comment (MohammedHasan-99) [inline css/home.css]: why did you remove this?

**Review by MohammedHasan-99 — CHANGES_REQUESTED** [review]

**Review by MohammedHasan-99 — APPROVED** [review]

**Review by MMansour87 — CHANGES_REQUESTED** [review]
Where the is the task id `<lable>` element in the right section of the task
we need to create it and assign its content with id value in local storage array
I need to refer to the array

**Review by Doaa-Abdulsalam — APPROVED** [review]

**Review by MMansour87 — APPROVED** [review]
I will do the id <lable> as it related to my issue

**Pull request merged**